promql/promqltest/testdata/native_histograms.test

```diff
@@ -1373,3 +1373,14 @@ eval instant at 1m histogram_fraction(-Inf, +Inf, histogram_nan)
     expect info msg: PromQL info: input to histogram_fraction has NaN observations, which are excluded from all fractions for metric name "histogram_nan"
     {case="100% NaNs"} 0.0
     {case="20% NaNs"} 0.8
+
+clear
+
+# Carefully chosen interval and range so that the zero point of the count is inside the
+# interpolation period.
+load 1m
+  metric {{schema:0 count:15.0 sum:25.0 buckets:[5 10]}} {{schema:0 count:2490.0 sum:75.0 buckets:[15 2475]}}x55
+
+eval instant at 55m increase(metric[90m])
```

Comment:
Here the last sample in the range coincides with the boundary of the range. Let's change this to have something in between, e.g.
```suggestion
eval instant at 54m30s increase(metric[90m])
```

(Spoiler: The values below change, but somehow the buckets still sum up to the count.)